pkg/asset/machines/libvirt/machines.go

```diff
@@ -73,7 +73,7 @@ func provider(platform *types.LibvirtPlatform, name string) *libvirtprovider.Lib
 			PoolName:     "default",
 			BaseVolumeID: "/var/lib/libvirt/images/coreos_base",
```

Comment:
this needs to change to match the https://github.com/openshift/installer/pull/660/files#diff-3051f6cd2c80dd39db1934750a7838acR2

Reply:
> this needs to change to match the https://github.com/openshift/installer/pull/660/files#diff-3051f6cd2c80dd39db1934750a7838acR2

Done with 80bb4ed -> bca7e1e, which also rebases onto master around #657.